hdlbits/035_always_casez/README.md

```diff
@@ -0,0 +1 @@
+Este programa cria um "priority encoder" utilizando a expressão casez. Essa expressão permite analisar um bit específico e ignorar bits ao avaliar cada caso.
```

Comment:
Por favor, escreva _priority encoder_ em itálico ao invés de entre "aspas". Coloque também palavras chaves com fonte fixa, por exemplo, `casez`

Reply:
Inclua um comentário sobre a necessidade de se usar Verilog 2001 para `casez`